CONTRIBUTING.md

```diff
@@ -77,9 +78,9 @@ The default test script will do the following: lint with ESLint -> unit tests wi
 
 ACPaaS UI is supplied as a single library on NPM [@acpaas-ui/react-components](https://www.npmjs.com/package/@acpaas-ui/react-components).
 
-It is released once a week, or more often if there are high priority changes. 
+It is released once a week, or more often if there are high priority changes.
 
-If your PR has been merged but not yet released, and it is high priority, please ping the [#acpaas-ui-react channel][acpaas-ui-react-slack] to ask for an out-of-band release. However, before you do so, container whether you can use a [local install or the npm link command](https://medium.com/@the1mills/how-to-test-your-npm-module-without-publishing-it-every-5-minutes-1c4cb4b369be) instead, or whether you can publish your own development snapshot package under your personal scope on NPM.
+If your PR has been merged but not yet released, and it is high priority, please ping the [#acpaas-ui-react channel][acpaas-ui-react-slack] to ask for an out-of-band release ( If you're not yet a member of our DigAnt Café slack community, you can easily [join here][digantcafe-slack].). However, before you do so, container whether you can use a [local install or the npm link command](https://medium.com/@the1mills/how-to-test-your-npm-module-without-publishing-it-every-5-minutes-1c4cb4b369be) instead, or whether you can publish your own development snapshot package under your personal scope on NPM.
```

Comment:
Not sure why `If you're not yet a ...` is set in parentheses and not just a separate sentence like in the README file.
Also the sentence `However, before you do so, container whether you can ...` seems a bit odd.